lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -14,7 +14,7 @@ class ShopifyDomainNotFound < StandardError; end
       rescue_from ActiveResource::UnauthorizedAccess, :with => :close_session
     end
 
-    def shopify_session
+    def manage_shopify_session
```

Comment:
Why did you change this methods name as well? This isnt really even a management method more of a `ensure_shopify_session` however I think it would be best to leave it as it was

Reply:
I was ultimately trying to free up the `shop_session` so that we could replace the `@shop_session` with `@shopify_session`. `@shop_session` made sense when `shopify_app` was limited to just `Shop` tokens - but now that we need to support per-user tokens, calling the Shopify session `@shop_session` isn't really correct. 

I think the `shopify_session` function is poorly named - it's really about setting up the `shop_session` context. But agree that `manage_shopify_session` is also poorly named. I can revert and you guys can decide if you would like to rename in the future.

Let me know what you think about `@shop_session`.

This does bring up another naming issue -